app/schemas.py

```diff
@@ -619,11 +633,12 @@ def handle_template_merge(self, in_data):
 
 
 class InvitedUserSchema(BaseSchema):
-    auth_type = field_for(models.InvitedUser, 'auth_type')
+    auth_type = field_for(models.InvitedUser, 'auth_type', default=EMAIL_AUTH_TYPE)
```

Comment:
Without the "default" parameter, "auth_type" was missing from the JSON created by this class.